### ISSUE TITLE
Feature request: use Maven's logging when invoked from the mvn plugin

### ISSUE THREAD

**Issue by daveloyall** [randomnoun/java-to-graphviz#1]
Here is a snippet from my maven console.

```
[INFO] Creating graphviz diagrams
[INFO] Reading D:\projects\workspace-2021-09\etltools\src\main\java\us\ne\state\ocio\tools\filemanagement\ArchiveItStatus.java
log4j:WARN No appenders could be found for logger (com.randomnoun.build.javaToGraphviz.astToDag.CommentExtractor).
log4j:WARN Please initialize the log4j system properly.
log4j:WARN See http://logging.apache.org/log4j/1.2/faq.html#noconfig for more info.
[INFO] Writing us\ne\state\ocio\tools\filemanagement/doc-files/ArchiveItStatus.dot
```

The first two lines are on `stdout`, the next three are on `stderr`, and the last one is on `stdout`.

Looks like the plugin uses `getLog(...)`. (Maybe that is an idiom in the maven plugin world?)

Looks like the dot language generator uses `Logger logger = Logger.getLogger(CommentExtractor.class);`.  (That's a log4j idiom.)

There is probably a way to conditionally use the Maven-style logging when invoked by Maven, but I did not research it.

**Comment by knoxg** [comment]
Yeah should be able to funnel log4j logs into mvn logs or vice versa, will take a look

**Comment by knoxg** [comment]
log4j is now initialised inside the maven plugin; depending on the verbose flag to the plugin it will either log at debug or info level.

Have released a new 1.0.4 version of java-to-graphviz-maven-plugin containing these changes & the latest java-to-graphviz jar
